src/main/java/springboot/Model/ItemList.java

```diff
@@ -2,34 +2,35 @@
 
 import javax.persistence.*;
 import java.util.ArrayList;
+import java.util.List;
 
-//@Entity
+@Entity
 public class ItemList {
```

Comment:
I'm thinking we could probably get rid of itemlist

Reply:
I agree with this

Reply:
It's already gone